File: app/src/main/java/com/example/myapplication/models/animemodels/AnimeGenreAndSearchResultModel.java
package com.example.myapplication.models.animemodels;

import java.util.List;

import lombok.Data;

@Data
public class AnimeGenreAndSearchResultModel {
    private AnimeSearchResult animeSearchResults;
    private AnimeSearchResultNew animeSearchResultsNew;
    private AnimeGenreResult animeGenreResults;

    @Data
    public class AnimeSearchResult {
        private String animeTitle;
        private String animeThumb;
        private String animeDetailURL;
        private String animeStatus;
        private String animeType;
    }

    @Data
    public class AnimeSearchResultNew {
        private String animeTitle;
        private String animeThumb;
        private String animeDetailURL;
        private String animeGenre;
        private String animeRating;
    }

    @Data
    public class AnimeGenreResult {
        private String genreTitle;
        private String genreURL;
    }
}

File: app/src/main/java/com/example/myapplication/models/animemodels/AnimeGenreAndSearchResultModel.kt
package com.example.myapplication.models.animemodels

data class AnimeGenreAndSearchResultModel(
        var animeSearchResults: AnimeSearchResult? = null,
        var animeGenreResults: AnimeGenreResult? = null
) {
    data class AnimeSearchResult(
            var animeTitle: String? = null,
            var animeThumb: String? = null,
            var animeDetailURL: String? = null,
            var animeStatus: String? = null,
            var animeType: String? = null
    )

    data class AnimeGenreResult(
            var genreTitle: String? = null,
            var genreURL: String? = null
    )
}
